doc/lua_api.txt

```diff
@@ -4010,7 +4012,9 @@ Definition tables
         ^ node placement (i.e. schematics and VoxelManip) or air nodes. If
         ^ return true the node is not flooded, but on_flood callback will
         ^ most likely be called over and over again every liquid update
-        ^ interval. Default: nil ]]
+        ^ interval. Note: liquid-type nodes cannot themselves be floodable, else
```

Comment:
If I understand you right, you're saying the engine does indeed honor the floodable property for nodes also defined as liquids? I did not observe behavior consistent with that, myself.

Sorry for the learner's question, but what does 'on a newline' mean? Should I replace what I wrote there, or put your addition underneath, or ...?